src/progeny/__init__.py

```diff
@@ -17,31 +33,25 @@ def __init__(cls, name, bases, dct):
         if '__progeny_key__' not in dct.keys():
             cls.__progeny_key__ = cls._get_progeny_key()
 
+    @property
+    def progeny(self):
```

Comment:
would progeny classes (e.g. `Alpha`, `Bravo`, etc. in the tests) also have a `.progeny`?

Reply:
to answer my own question: they do, which I don't love. I'm not sure if it'd be more confusing if they didn't have it nor how much effort. no strong opinion (and definitely not a blocker), just food for thought

Reply:
They should, yes. That behavior is not explicitly tested so I'm not going to promise that it works, but that was my intention.